types/part_set.go

```diff
@@ -88,19 +88,19 @@ func NewPartSetFromData(data []byte, partSize int) *PartSet {
 	// divide data into 4kb parts.
 	total := (len(data) + partSize - 1) / partSize
 	parts := make([]*Part, total)
-	parts_ := make([]merkle.Hasher, total)
+	parts_ := make([][]byte, total)
```

Comment:
While you are at it, can you rename `parts_` to something more meaningful? E.g. `partsData` or `partsBytes`?